### PR TITLE
Change naming and fix bugs in DTO module, add live data generator to timeseries module.

### PR DESCRIPTION
**Pull request by erlendcognite**
- Bug in DTOs: to_pandas() would mutate internal representation of data for certain response objects.
- Add live data generator.
- Breaking Changes:
1. rename data_objects.py -> dto.py
2. rename timeseries paremeter to 'time_series' when input is list of time series and 'name' when input is single time series.
3. rename AssetDTO -> Asset
4. rename EventDTO -> Event
5. rename TimeSeriesDTO -> TimeSeries
6. rename RawRowDTO -> RawRow
7. rename DatapointDTO -> Datapoint

### CODE REVIEW

File: cognite/v05/timeseries.py [hunk 1]
```diff
@@ -150,8 +151,8 @@ def _get_datapoints_helper(timeseries, aggregates=None, granularity=None, start=
         list of datapoints: A list containing datapoint dicts.
     '''
     api_key, project = kwargs.get('api_key'), kwargs.get('project')
-    timeseries = timeseries.replace('/', '%2F')
-    url = config.get_base_url(api_version=0.5) + '/projects/{}/timeseries/data/{}'.format(project, timeseries)
+    name = name.replace('/', '%2F')
```
Comment (stianlagstad) [inline cognite/v05/timeseries.py]: Is there some sort of urlencode-function you could use here instead? and elsewhere

File: cognite/v04/dto.py [hunk 2]
```diff
@@ -212,7 +213,7 @@ def to_json(self):
 
     def to_pandas(self):
         '''Returns data as a pandas dataframe'''
-        items = self.internal_representation['data']['items']
+        items = deepcopy(self.internal_representation['data']['items'])
```
Comment (stianlagstad) [inline cognite/v04/dto.py]: Does it make sense to write a test that proves that the mutation bug is fixed?

File: cognite/v05/timeseries.py (hunk 1 above)
Comment (erlendcognite) [reply cognite/v05/timeseries.py]: Yes, there is. Will use.

File: cognite/v04/dto.py (hunk 2 above)
Comment (erlendcognite) [reply cognite/v04/dto.py]: Good idea - makes perfect sense.

**Pull request merged**